docs/getting-started/launching-one-node/crypto-materials.md

```diff
@@ -9,3 +9,66 @@ title: Creating Crypto Material
  SPDX-License-Identifier: CC-BY-4.0
  -->
 
+### Crypto materials configuration
+
+Hyperledger Orion uses public key cryptography from user side, for user authentication and transaction Non-Repudiation. From server side, Orion uses PKI to sign and verify all server responses to clients. 
+
+Thus, each Orion client needs certificate/key pair to sign each query and transaction he submits to server, and server has to have its own certificate/key pair to sign each response to client.
+
+From another side, each Orion server has to keep each client certificate to validate client signatures and each client has to have access to server certificate to validate servers signatures. Both server and client need access to CA certificate to validate new client/server certificates.
+
+So, to start Orion new server, each server has to have following:
+1. CA certificate (we support multiple CAs, including multilevel CAs).
+   * `caConfig.rootCACertsPath` 
+2. Its own certificate, sign by CA from above and corresponding private key.
+   * `server.identity.certificatePath` and `server.identity.keyPath`
+3. Default administrative user certificate, usually denoted as `admin` certificate.
+   * `admin.certificatePath`
+
+To work with Orion server, each client needs:
+1. CA certificate.
+```go 
+&config.ConnectionConfig{     
+   RootCAs: []string{
+      "./crypto/CA/CA.pem",
+   },
+}
+```
+2. Its own certificate, sign by CA from above and corresponding private key.
+```go
+&config.SessionConfig{
+	UserConfig: &config.UserConfig{
+		UserID:         userID,
+		CertPath:       "./crypto/" + userID + "/" + userID + ".pem",
+		PrivateKeyPath: "./crypto/" + userID + "/" + userID + ".key",
+	},
+}
+```
+3. Server certificate can be accessed using node config query, [here](../queries/curl/node-config) and [here](../queries/gosdk/node-config).
+
+> For now, we ignore TLS configuration and location of TLS certificates, both servers and clients.
+
+### Crypto materials generation
+To generate new crypto materials, we use `scripts/cryptoGen.sh`. As first argument, it accepts absolute path to folder to store the generated files. 
+
+Running it with only first argument, will generate minimal set of crypto materials includes 3 sets of certificates, totally 4 certificates
+
+1. `admin` and `user` - for db users
+2. `server` - for server node
+3. CA - for Certificate Authority 
+
+To create minimal set of cryptographic materials, you can run:
+`./scripts/cryptoGen.sh deployment`
+
+This run will generate 4 folders under `deployment/crypto`:
+1. CA
+2. server
+3. admin
+4. user
+
+If you need more certificates (for extra client or extra server nodes), they can be specified as extra arguments: 
+
+`./scripts/cryptoGen.sh deployment <extra users>`
```

Comment:
`./scripts/cryptoGen.sh deployment <extra users>`
for example
`./scripts/cryptoGen.sh deployment alice bob`